examples/IrisClassification/iris_handler.py

```diff
@@ -25,15 +29,53 @@ def preprocess(self, data):
 
         :return: output - Preprocessed input
         """
+        from mlflow_torchserve.SignatureValidator import SignatureValidator
 
-        input_data_str = data[0].get("data")
-        if input_data_str is None:
-            input_data_str = data[0].get("body")
+        data = json.loads(data[0]["data"].decode("utf-8"))
+        df = pd.DataFrame(data)
 
-        input_data = input_data_str.decode("utf-8")
-        input_tensor = torch.Tensor(ast.literal_eval(input_data))
+        SignatureValidator(model_meta=self.mlmodel)._enforce_schema(
+            df, self.mlmodel.get_input_schema()
+        )
```

Comment:
Can we just use [`mlflow._enforce_schema`](https://github.com/mlflow/mlflow/blob/25379a3c1970c6a5a65441b5b813965ddab50d1f/mlflow/pyfunc/__init__.py#L484)?

Reply:
done